src/AutoVizuA11y.tsx

```diff
@@ -134,6 +147,15 @@ const AutoVizuA11y = ({
 
 	let componentId = useAutoId();
 
+	useEffect(() => {
+		if (visibleShortcutGuide) {
+			let modalContent = shortcutGuideRef.current!.getElementsByClassName(
+				"shortcut-guide__container",
```

Comment:
Possible improvement: shouldn't we have a collection of constants (or a single object) with these classes so that we could re-use them?